packages/core/src/components/map/Map.tsx

```diff
@@ -63,15 +63,14 @@ interface PrivateMapProps<Type extends Area> extends MapProps<Type> {
   areas: readonly Type[]
 }
 
-const defaultProps: MapProps<Area> = {
-  style: {},
-  color: '#ccc',
-  showTooltip: true,
-  hoverable: true,
-  zoomable: true
-}
-
-function Map<Type extends Area>(props: PrivateMapProps<Type>) {
+const Map = <Type extends Area>({
+  style = {},
+  color = '#ccc',
+  showTooltip = true,
+  hoverable = true,
+  zoomable = true,
+  ...props
+}: PrivateMapProps<Type>) => {
   test.rerenders()
 
```

Comment:
This is a good solution! But since we are now destructuring some props, we may as well destructure all props and reference them without `props.`. I think we can do this after `test.rerenders()` by just doing something like:

```ts
const { 
  style = {},
  color = '#ccc',
  onClick,
  ...
} = props
```

This should make it a little cleaner (;